data/geany.glade

```diff
@@ -6321,6 +6321,7 @@
               <object class="GtkMenuBar" id="menubar1">
                 <property name="visible">True</property>
                 <property name="can_focus">False</property>
+                <signal name="deactivate" handler="on_menubar1_deactivate" after="yes" swapped="no"/>
```

Comment:
I've experimented a bit, but can't find the difference between `activate` and `select`. GTK docs aren't clear about this too.